catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/DeleteOperations.java

```diff
@@ -473,4 +529,29 @@ private boolean isStorageAvailable(StorageProvider storageProvider) {
         }
         return true;
     }
+
+    private void logFailedQueryInfo(DeleteRequest deleteRequest, QueryResponse query) {
+        if (LOGGER.isDebugEnabled()) {
+            final String attributeName = deleteRequest.getAttributeName();
+            Set<String> queryResults = query.getResults()
+                    .stream()
+                    .map(Result::getMetacard)
+                    .map(m -> m.getAttribute(attributeName))
+                    .filter(Objects::nonNull)
+                    .map(Attribute::getValue)
+                    .filter(Objects::nonNull)
+                    .map(Object::toString)
+                    .collect(Collectors.toSet());
+
+            LOGGER.debug(
```

Comment:
Well, we certainly try to avoid using multiple log statements at the same level at the same point. I'm not entirely opposed to it, in this case, because they seem like log messages with a lot of info in them. Unless someone has a better idea, I'd say it is fine.